src/secure_random.cr

```diff
@@ -1,20 +1,64 @@
 require "base64"
 require "openssl/lib_crypto"
 
+# The SecureRandom module is an interface for creating secure random values in different formats, 
+# including Base64 string, hexadecimal string, uuid and others. It uses LibCrypto(OpenSSL) as the
```

Comment:
Base64 string sounds silly, the entire point of Base64 is to get something in ASCII. UUID should be capitalized, but I'm not even sure we should list all the formats here, it's quite redundant. Also missing space before (OpenSSL). And how about "It uses the RNG (random number generator) of libcrypto (OpenSSL)."